versions.tf

```diff
@@ -19,8 +19,10 @@ terraform {
       version = ">= 1.3"
     }
     kubernetes = {
+      # Version 2.0 has breaking changes, this module needs conversion to use it.
```

Comment:
@Nuru  #93 has the appropriate v2 provider changes.

Reply:
#93 has breaking changes, and I'm not sure how much we can validate about the changes to the handling of `KUBECONFIG`. So I would like to release this PR first to have a proven module that does not require users to pin the Kubernetes provider themselves. Then we can release #93 later after fully considering the changes.

Reply:
Sure. I hadn't seen your previous feedback before i left this one.